data/trie/patriciaMerkleTrie.go

```diff
@@ -6,32 +6,40 @@ import (
 	"sync"
 
 	"github.com/ElrondNetwork/elrond-go/data"
+	"github.com/ElrondNetwork/elrond-go/data/trie/evictionWaitingList"
 	"github.com/ElrondNetwork/elrond-go/hashing"
 	"github.com/ElrondNetwork/elrond-go/marshal"
+	"github.com/ElrondNetwork/elrond-go/storage"
+	"github.com/ElrondNetwork/elrond-go/storage/memorydb"
 )
 
 const (
 	extension = iota
 	leaf
 	branch
 )
+const evictionCacheSize = 100
```

Comment:
would it make sense to move this also to config.toml ?

Reply:
done.